src/list-sorting/index.js

```diff
@@ -1,3 +1,21 @@
-function listSorting(needle, haystack) {}
+function listSorting(needle, haystack) {
+    if (typeof haystack[0] != "object"){
```

Comment:
Prefer `Array.isArray` when checking if an object is an array.

Reply:
Noted